--- _build/default/theories/extract.ml	2020-08-27 16:34:09.889562027 +0100
+++ _build/default/src/extract.ml	2020-08-27 16:34:10.365528985 +0100
@@ -15929,13 +15929,13 @@
 
   (** val executable_host_instance : executable_host **)
 
-  let executable_host_instance =
-    { Coq__9.host_monad = (Obj.magic host_monad); host_apply =
+  let executable_host_instance : executable_host =
+    { Coq__3.host_monad = (Obj.magic host_monad); Coq__3.host_apply =
       (Obj.magic H.host_apply) }
 
   (** val host_functor : __ H.host_event functor0 **)
 
-  let host_functor =
+  let host_functor : __ H.host_event functor0 =
     functor_Monad host_monad
  end
 
@@ -17063,7 +17063,7 @@
 
   (** val monad_functor : __ M.monad functor0 **)
 
-  let monad_functor =
+  let monad_functor : __ M.monad functor0 =
     functor_Monad monad_monad
  end
 
